app/src/main/java/com/nexenio/bleindoorpositioningdemo/ui/beaconview/chart/BeaconChartFragment.java

```diff
@@ -23,11 +26,15 @@
 
 import java.util.ArrayList;
 import java.util.UUID;
+import java.util.concurrent.TimeUnit;
 
 public class BeaconChartFragment extends BeaconViewFragment {
 
     private BeaconChart beaconChart;
 
+    protected BeaconManager beaconManager = BeaconManager.getInstance();
+    private Boolean filterView = true;
```

Comment:
You didn't provide a way to adjust this value. Keep in mind that changing it after the layout has been inflated will have no effect, so you may want to do that in the constructor.